config/locales/en.yml

```diff
@@ -553,8 +553,7 @@ en:
         edit:
           page_title: Previous proceedings
           section: Previous proceedings
-          heading: Have any of the children in this application been involved in another family court case?
```

Comment:
Revert back to how it was before. See comment above re: child_protection_cases

Reply:
I've moved hard coded text to the locale translation file, apart from the home page. As well, made the requested fixes. I've also removed redundant classes to style the h2 tag and used the one already available.

Reply:
Made the requested fixes. Removed redundant classes to style the h2 tag, and corrected a minor typo on the front page.